src/styles/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 16px 0;
+  color: var(--main-color);
+}
+
+.card__product-code {
+  font-weight: 400;
```

Comment:
400 is default value - no need to write it manually
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight